tests/manage.py

```diff
@@ -3,6 +3,13 @@
 import sys
 import warnings
 
+import django
+if django.VERSION[:2] == (1, 6):
+    # This is only necessary for Django 1.6
+    from django.contrib.auth.tests import custom_user
+    custom_user.AbstractUser._meta.local_many_to_many = []
+    custom_user.PermissionsMixin._meta.local_many_to_many = []
```

Comment:
Is this related to https://github.com/django/django/blob/master/django/contrib/auth/tests/custom_user.py#L80-L98?